### PR TITLE
fix: remove splitchunks

### LINKED ISSUE #404: [Feature]: Init command's splitChunks caching group generation per provided entry
**Do you want to request a *feature* or report a *bug*?**

feature

**What is the current behavior?**
it doesnt output caching group per entry provided

**If the current behavior is a bug, please provide the steps to reproduce.**
after #356 is merged
use `webpack-cli init` and provide multiple entries
Observe splitChunks plugin declaration doesnt add caching group per entry

**What is the expected behavior?**
splitChunks caching groups added per entry

**If this is a feature request, what is motivation or use case for changing the behavior?**
TBD @ev1stensberg 

**Please paste the results of `webpack-cli info` here, and mention other relevant information such as programming language.**

*NOTE*
When trying to initally set up regexp generation i bumped into a case where user might provide two entries from within same directory when doing simple config generations e.g. for entries user types:
`homepage,blog`

and for entry paths:
```
./src/homepage
./src/blog
```

In this case regexp for catching files in splitChunks will be the same for both entries if entries are
```
./src/homepage.js
./src/blog.js
```

and not directories with a nested `index.js` file.
```
./src/homepage/index.js
./src/blog/index.js
```

### PR DESCRIPTION
**Pull request by rishabh3112**
**What kind of change does this PR introduce?**
closes #404
remove adding splichunk config, using default from webpack by default.

**Did you add tests for your changes?**
WIll update along the way

**If relevant, did you update the documentation?**
No

**Summary**
removed addition of splitchunks

**Does this PR introduce a breaking change?**
No

**Other information**
No

### CODE REVIEW

**Review by alexander-akait — DISMISSED** [review]
Let's remove `TerserPlugin` too in separate PR, we should always use built-in minimizer

**Review by anshumanv — DISMISSED** [review]
LGTM, wip on terser

**Review by snitin315 — APPROVED** [review]

File: packages/generators/src/init-generator.ts [hunk 1]
```diff
@@ -233,7 +224,6 @@ export default class InitGenerator extends CustomGenerator {
         this.configuration.config.topScope.push(tooltip.terser(), "const TerserPlugin = require('terser-webpack-plugin');", '\n');
 
         // Chunksplitting
-        this.configuration.config.webpackOptions.optimization = getDefaultOptimization(this.usingDefaults);
```
[the remark this answers is not in the document]
Comment (rishabh3112) [reply packages/generators/src/init-generator.ts:236]: Done

**Comment by rishabh3112** [comment]
> Let's remove TerserPlugin too in separate PR, we should always use built-in minimizer

terserPlugin was redundant after this PR, so removed.

**Review by snitin315 — APPROVED** [review]

**Review by alexander-akait — APPROVED** [review]

**Pull request merged**